packages/components/src/components/carousel/carousel.ts

```diff
@@ -84,6 +84,12 @@ export default class SdCarousel extends SolidElement {
    */
```

Comment:
unfortunately I cannot tag line 56 here but I found a strange behavior in Storybook on the variant property: 
![image](https://github.com/user-attachments/assets/1a02e4ee-3731-4030-9a10-d37bdc23e9ee)
shouldn't this be a string property instead of number? maybe the term `number` is reserved here... :/

Reply:
I accidentally introduced a bug when I was cleaning up my code. This should be working now.

Reply:
I think youre comment was towards my other conversation above right?

I will open up a new bug ticket as I found something else.